examples/rbac_with_domains_model.conf

```diff
@@ -0,0 +1,14 @@
+[request_definition]
+r = sub, dom, obj, act
+
+[policy_definition]
+p = sub, dom, obj, act
+
+[role_definition]
+g = _, _, _
+
+[policy_effect]
+e = some(where (p.eft == allow))
+
+[matchers]
+m = g(r.sub, p.sub, r.dom) && r.dom == p.dom && r.obj == p.obj && r.act == p.act
```

Comment:
Please convert this into yaml

Reply:
@hackerchai I think model doesn't need to be YAML format?

Reply:
@hsluoyz You are right. I will close this.